### PR TITLE
upgrade gradle to 3.3, spring framework 4.3.6, and spring boot 1.5.1

### PR DESCRIPTION
**Pull request by zyndaras**
@daves125125

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -22,19 +22,20 @@ targetCompatibility = 1.8
...
-version = '0.0.12-SNAPSHOT'
+version = '0.0.13-SNAPSHOT'
 
 
 dependencyManagement {
     imports {
-        mavenBom 'io.spring.platform:platform-bom:Athens-RELEASE'
-        mavenBom 'org.springframework.cloud:spring-cloud-dependencies:Camden.RELEASE'
+        mavenBom 'io.spring.platform:platform-bom:Brussels-RC1'
+        mavenBom 'org.springframework.cloud:spring-cloud-dependencies:Camden.SR5'
```
Comment (daves125125) [inline build.gradle]: Is Dalston not out? Why the SR releases of Camden?

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Tue Oct 11 23:33:12 BST 2016
+#Tue Feb 07 12:30:34 GMT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.13-all.zip
```
Comment (daves125125) [inline gradle/wrapper/gradle-wrapper.properties]: Nice

Comment (daves125125) [inline gradle/wrapper/gradle-wrapper.properties]: Nice

File: build.gradle [hunk 3]
```diff
@@ -49,7 +50,7 @@ dependencies {
     compile 'javax.inject:javax.inject:1'
     compile 'org.reflections:reflections:0.9.10'
 
-    testCompile 'ch.qos.logback:logback-classic:1.1.7'
+    testCompile 'ch.qos.logback:logback-classic:1.1.9'
```
Comment (daves125125) [inline build.gradle]: Was this causing issues?

Comment (zyndaras) [inline build.gradle]: it showed this warning:
`SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/zyndaras/.gradle/caches/modules-2/files-2.1/ch.qos.logback/logback-classic/1.1.7/9865cf6994f9ff13fce0bf93f2054ef6c65bb462/logback-classic-1.1.7.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/zyndaras/.gradle/caches/modules-2/files-2.1/ch.qos.logback/logback-classic/1.1.9/978cd9fbb43b7abed6379d7b02de052d216e30fc/logback-classic-1.1.9.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [ch.qos.logback.classic.util.ContextSelectorStaticBinder]`

which disappeared when upgrading.

File: build.gradle (hunk 1 above)
Comment (zyndaras) [inline build.gradle]: No particular reason, the Camden SR5 keeps certain dependencies in RELEASE whereas Dalston M1 brings milestone versions, for example:
spring-cloud-consul-config 1.1.2.RELEASE vs 1.2.0.M1

File: build.gradle (hunk 3 above)
Comment (daves125125) [inline build.gradle]: Sounds like that version is already on the claspath then (probably transitive from spring cloud deps), so maybe you can get rid of it entirely.

**Pull request merged**